crm_lead_code/__manifest__.py

```diff
@@ -22,9 +21,10 @@
 
 {
     "name": "Sequential Code for Leads / Opportunities",
-    "version": "10.0.1.0.0",
+    "version": "12.0.1.0.0",
     "category": "Customer Relationship Management",
-    "author": "OdooMRP team, "
+    "author": "MuK IT GmbH, "
```

Comment:
This migration doesn't grant you co-authorship. Please include yourself in contributors, not here.